matlab/schemas/+meso/SegmentationManual.m

```diff
@@ -5,13 +5,81 @@
 
 
 classdef SegmentationManual < dj.Computed
+    
+    properties
+        popRel  = meso.SummaryImagesAverage
+    end
+    
+    methods(Access=protected)
+        function makeTuples(self, key)
```

Comment:
Don't really know whether this is ok, but I guess it matches the reso MannualSegmentation code